app/assets/javascripts/code/services/_draw_session.js

```diff
@@ -61,7 +61,21 @@ export const drawSession = (
       return session.markers;
     },
 
-    calculateHeatLevel: function(value) {
+    drawMobileSessionStartPoint: function(session) {
+      if(!session || !sensors.anySelected()){
+        return;
+      }
+      this.undoDraw(session);
+
+      session.markers = [];
+      var markerOptions = {title: session.title, zIndex: 0};
+      var lngLatObject = { longitude: session.starting_longitude, latitude: session.starting_latitude }
```

Comment:
We should be using camelCase for Js variables

Reply:
They come from rails part, that's why they are snake, but I guess I can change them there and have camel here.

Reply:
For now it's fine. We should use serializers on the rails side doing that automatically